app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
```

Comment:
__[staging][phpmd]__
* Avoid variables with short names like $a2. Configured minimum length is 3.
* Avoid variables with short names like $b2. Configured minimum length is 3.
* Avoid unused local variables such as '$a2'.
* Avoid unused local variables such as '$b2'.